src/mito2/src/gc.rs

```diff
@@ -316,38 +316,42 @@ impl LocalGcWorker {
         let region_id = manifest.metadata.region_id;
         let current_files = &manifest.files;
 
+        let in_manifest_file_cnt = current_files.len();
```

Comment:
Seems most of the following new variables are only used in debug logging, better move them into the `debug!`.